rpa-home/modules/ROOT/pages/automation-security-securesessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Secure Sessions
+
+In RPA automations, a bot simulates a human user. The bot logs in to a system, sees the screen, and can interact with the UI via key strokes and mouse actions. That way, the bot can even interact with legacy systems, that do not provide an API.
+
+The bot logs in to the computer, where it is installed. But, of course, it does not sit on a chair in front of the desktop. Nevertheless, if you would enter the room where the computer is, you could see what the bot is doing: The screen would show the open applications, the mouse would  move around and typing would appear in forms. In MuleSoft RPA, we call this a _Desktop Session_. To prevent unauthorized data access or manipulation, you would need to guard the physical room.
+
+Or you could use what we call _Secure Sessions_.
+
+Secure Sessions are hidden Desktop Sessions. They run in the background and only the bot can access them. An additional advantage of the Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
+
+== General Considerations
+
+* At the same time, a bot can only run in Desktop or in Secure Session mode.
+* You can not switch the session mode while configurations are deployed on the bot.
+* Run configurations specify a session mode, too.
+* You can only deploy configurations to a bot that is in the same session mode.
+* You can specify the login data for the bot and the maximum number of Secure Sessions that the bot may run in parallel in the RPA Bot Configurator or in the RPA Manager.
+** The maximum number of Secure Sessions set and enabled in the RPA Bot Configurator overwrites the same setting of RPA Manager.
+* Desktop Sessions:
+** The specified user must be logged in to this session to deploy a run configuration. 
+** Do not close or minimize the console session while the bot executes a process.
```

Comment:
Our style is to make bullet list items parallel. This bullet is an imperative instruction, but the previous one is just a requirement. Let me know if this is unclear.